### PR TITLE
Add target='_blank' for redirects of dashboard link

### LINKED ISSUE #5192: Dashboard link redirects on same tab
When  I click the link for the dashboard it redirects me to the dashboard but on the same tab instead of a new one, leaving me with no access to the notebook. I think it would be good that by default it redirects to a new tab. 

```
from dask.distributed import Client
client=Client()
client
```

redirect_link

Distributed version 2021.07.2 (tried on main too)
Browser Firefox 90.0.2 and Safari 14.1.1

### PR DESCRIPTION
**Pull request by ncclementi**
- [x] Closes  #5192
- [ ] Tests added / passed
- [x] Passes `black distributed` / `flake8 distributed` / `isort distributed`

This will cause a redirect to a new tab every time a dashboard link is clicked on the html repr displayed in the notebooks. I'm still missing one case which is the `LocalCluster`link, but I couldn't find where is the template with the `href` to the dashboard for the case of LocalCLuster. For reference, I'm talking about this link:

Screen Shot 2021-08-19 at 4 22 48 PM

cc: @jacobtomlinson

### CODE REVIEW

File: distributed/widgets/templates/cluster.html.j2 [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <table style="width: 100%; text-align: left;">
             <tr>
                 <td style="text-align: left;">
-                    <strong>Dashboard:</strong> <a href="{{ dashboard_link }}">{{ dashboard_link }}</a>
+                    <strong>Dashboard:</strong> <a href="{{ dashboard_link }} target="_blank"">{{ dashboard_link }}</a>
```
Comment (jrbourbeau) [inline distributed/widgets/templates/cluster.html.j2:10]: Thanks @ncclementi! 

> I'm still missing one case which is the LocalClusterlink, but I couldn't find where is the template with the href to the dashboard for the case of LocalCLuster

I think this is where the `LocalCluster` dashboard link comes from. I see there are two double-quotes at the end, but I think we only want one. Maybe that's what's causing the link to not open in a separate tab

Comment (jrbourbeau) [reply distributed/widgets/templates/cluster.html.j2:10]: Specifically, I mean this

```suggestion
                    <strong>Dashboard:</strong> <a href="{{ dashboard_link }}" target="_blank">{{ dashboard_link }}</a>
```

Comment (ncclementi) [reply distributed/widgets/templates/cluster.html.j2:10]: Great catch @jrbourbeau, thanks!

**Review by jrbourbeau — APPROVED** [review]
Thanks @ncclementi! This LGTM, but I'll let @jacobtomlinson have the final sign off on this one

**Review by jacobtomlinson — APPROVED** [review]

**Comment by jacobtomlinson** [comment]
Looks good thanks!

**Comment by jacobtomlinson** [comment]
> I'm still missing one case which is the LocalCluster

`LocalCluster` is a subclass of `Cluster` so the `cluster.html.j2` template will be used, so this PR should solve that too.

**Pull request merged**